app/app.js

```diff
@@ -9,12 +10,38 @@ app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true}));
 app.use(express.static(path.join(__dirname, '..', 'public')));
 
-/*
-=====
-OTHER ROUTES GO HERE
-=====
-*/
+app.get('/api/formdata', function(req, res) {  // Will return all entries
+  Entry.find()
+  .then(function(entry) {
+    res.status(200).json(entry)
+  })
+  // curl --request GET 'localhost:3000/api/formdata'
+})
 
+app.post('/api/formdata', function(req, res) {
+  var entry = new Entry({
+    userId: req.body.userId,
+    datetime: req.body.datetime,
+    type: req.body.type,
+    ingredients: req.body.ingredients,
+    sleepDuration: req.body.sleepDuration,
+    sleepQuality: req.body.sleepQuality,
+    exerciseDuration: req.body.exerciseDuration,
+    exerciseIntensity: req.body.exerciseIntensity,
+    waterAmount: req.body.waterAmount,
+    physicalScore: req.body.physicalScore,
+    emotionalScore: req.body.emotionalScore,
+    physicalTags: req.body.physicalTags,
+    emotionalTags: req.body.emotionalTags
+  })
+  entry.save(function (err, post) {
+    if (err) {
+      return 'There was an err', err
+    }
+    res.status(201).json(post)
```

Comment:
link this to a then block after the save promise:
return entry.save().then(()=>res.status(201).send('entry created'));   // we dont' need to send back the new entry data